packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -111,79 +89,87 @@ export class ProjectService {
         return project;
     }
 
-    async create(user: SessionUser, data: CreateProject): Promise<Project> {
+    async create(
+        user: SessionUser,
+        data: CreateProject,
+    ): Promise<{ jobUuid: string }> {
         if (user.ability.cannot('create', 'Project')) {
             throw new ForbiddenError();
         }
 
-        const jobUuid = await this.startJob(undefined);
+        const job: CreateJob = {
+            jobUuid: uuidv4(),
+            jobType: JobType.CREATE_PROJECT,
+            jobStatus: JobStatusType.STARTED,
+            projectUuid: undefined,
+            steps: [
```

Comment:
these steps still don't match the steps proposed by design. are we going to change it later, does that affect the current implementation ?

Reply:
Yes we'll fix this soon!